viewer/packages/camera-manager/src/ComboControls.ts

```diff
@@ -47,6 +47,7 @@ export class ComboControls extends EventDispatcher {
   public dampingFactor: number = 0.25;
   public dynamicTarget: boolean = true;
   public minDistance: number = 0.1;
+  public minZoomDistance: number = 0.1;
```

Comment:
Could this be set directly to the default value that is given in CameraManager, or is there a reason we would want them to be defined to separate values? Goes for all variables in this block

Reply:
Yeah, can be the same as default inside CameraManager